### ISSUE TITLE
Color customization does not work on iPad with home assistant app

### ISSUE THREAD

**Issue by RK62** [decompil3d/lovelace-hourly-weather#731]
**Checklist:**
- [X] I updated to the latest version available
- [X] I cleared the cache of my browser

**Release with the issue:**
Home Assistant 2008.08

**Last working release (if known):**
Home Assistant 2008.07

**Browser and Operating System:**
Home Assistant App on iPad

**Description of problem:**
The color adjustment is ignored on the iPad when using the home assistant app. Works perfectly on the iPhone or on the same iPad with Safari.

**Additional information:**
Releases: 
- Home Assistant 2024.8.2
- Home Assistant App 2024.7
- Hourly Weather Card: 6.1.0

Same problem as here? https://github.com/mlamberts78/weather-chart-card/issues/222

See Hardcopy: left side Safari, right side Home Assistant App
![IMG_20240820_184601-0273](https://github.com/user-attachments/assets/d28410e3-14d2-4b11-bf0a-34a2626577d9)

**Comment by decompil3d** [comment]
Writing myself some notes here. I'm not able to dive into this right now, but maybe someone else wants to take a stab.

Perhaps related to home-assistant/iOS#2908.

PR they suggested for the other card: https://github.com/mlamberts78/weather-chart-card/pull/223

Tl;dr: remove dynamic css styles. Use css vars in `styleMap` instead.

**Comment by RK62** [comment]
Works fine again with Home Assistant App 2024.8
